### ISSUE TITLE
Rename repo

### ISSUE THREAD

**Issue by dabreegster** [acteng/abstreet-to-atip#12]
The "abstreet to atip" name is historical, from before the project structure was well understood. This repo is used to prepare/generate data files needed by the ATIP frontend. How about "atip-data-prep" or "atip-importer" or "atip-input"? Thoughts, @robinlovelace-ate and @Pete-Y-CS?

**Comment by Robinlovelace** [comment]
:+1: to that. More ideas: "atip-route-generator". Of the ones you listed "atip-data-prep" is my favourite because  it does what it says on the tin, "importer" and "input" are less specific.

**Comment by Pete-Y-CS** [comment]
I agree, from the description you've given atip-data-prep seems most relevant. Maybe even atip-data-prep-scripts if we wanna be super specific

**Comment by dabreegster** [comment]
Thanks both! "route-generator" is too specific; this repo also has boundary-fixing scripts, and soon things unrelated to routing. I'll rename atip-data-prep
